docs/organizations/managing-security-and-risk.md

```diff
@@ -104,6 +104,37 @@ The following table defines item priorities and days to fix the underlying Codac
 | Medium                  | Medium        | 90            |
 | Minor                   | Low           | 120           |
 
+## Access management<!-- Rename this. We're not really only managing access, are we? -->
+
+Organization owners can assign the Security Manager role to organization members, as well as revoke it. Security managers share some permissions with organization owners:
```

Comment:
In particular, we should state here more clearly why organization owners may want to assign the Security Manager role to other people in their teams (or put into other words, why this is a useful feature).

Still dependent on the answers we get to the questions in the comments, for now it seems that the main advantage is that Security Managers will be able to access the Security and Risk dashboard.